docs/source/federated-types/federated-directives.mdx

```diff
@@ -657,3 +657,71 @@ If any of these requirements is not met, composition fails.
 
 </tbody>
 </table>
+
+## Security
+
+### `@authenticated`
+
+> ⚠️ **This directive is available in Apollo Federation 2.5 and later.**
+> ⚠️ **This is an [Enterprise feature](/router/enterprise-features) of the Apollo Router.** It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).</br>
+>
+> If your organization _doesn't_ currently have an Enterprise plan, you can test out this functionality by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
+
+```graphql
+directive @authenticated on
+    FIELD_DEFINITION
+  | OBJECT
+  | INTERFACE
+  | SCALAR
+  | ENUM
+```
+
+Indicates to composition that target element is accessible only to the authenticated supergraph users. For more granular access control see [`@requiresScope`](#requiresScope) directive below. See [JWT Authentication in the Apollo Router](/router/configuration/authn-jwt) for additional details.
```

Comment:
```suggestion
Indicates to composition that the target element is accessible only to the authenticated supergraph users. For more granular access control, see the [`@requiresScope`](#requiresScope) directive below. Refer to the [Apollo Router article](/router/configuration/authorization#authenticated) for additional details.
```

Links to the more detailed WIP page in Router docs. (See https://github.com/apollographql/router/pull/3449/files)